paasta_tools/cli/cmds/spark_run.py

```diff
@@ -413,6 +413,7 @@ def get_spark_config(
     volumes,
     access_key,
     secret_key,
+    session_token,
```

Comment:
should we make this optional? 
Otherwise, you will need to modify this line https://sourcegraph.yelpcorp.com/Yelp/paasta@31a946cfbf28b8b11382fc5c688bf64317fc05fc/-/blob/paasta_tools/cli/cmds/spark_run.py#L678

Reply:
Hm, isn't that file just a mirror of this code? It's modified on line `688` below.

Reply:
ah, you're right. some code are folded, and I misread the code.